### PR TITLE
refactor: add XML root tag

### LINKED ISSUE #77: Add root element to XML output recipe file
XML files that the expert system produces are formally invalid, because they miss an [XML root element](
https://en.wikipedia.org/wiki/Root_element). This can for instance be seen when you run the [`check-xml`](https://github.com/pre-commit/pre-commit-hooks#check-xml) of pre-commit:

```bash
pre-commit run check-xml -a
```

See also
https://github.com/ComPWA/expertsystem/blob/d4f1928ccaaa208f36f428d5dfdb8a94fa75fbbc/tox.ini#L42-L44

In addition, if there is no single (!) root tag, `xmltodict` cannot `unparse` the file.

### PR DESCRIPTION
**Pull request by redeboer**
Closes #77
The lack of a root tag made it impossible to *import* an XML recipe file with `xmltodict`.

Also adds a VSCode debug file that makes it easier to debug individual Python files.

_Follow-up to #98_
_Part of the PR series mentioned in https://github.com/ComPWA/expertsystem/issues/22#issuecomment-654052278_

### CODE REVIEW

File: particle_list.xml [hunk 1]
```diff
@@ -1,3695 +1,3623 @@
...
-      <Type>Strangeness</Type>
-      <Value>0</Value>
-    </QuantumNumber>
-    <QuantumNumber>
-      <Class>Int</Class>
-      <Type>BaryonNumber</Type>
-      <Value>0</Value>
-    </QuantumNumber>
-  </Particle>
-
```
Comment (redeboer) [inline particle_list.xml:142]: Let me add back these, they were removed by mistake

File: examples/python/D0ToKs0KpKm.xml [hunk 2]
```diff
@@ -1,417 +1,418 @@
 <?xml version="1.0" encoding="utf-8"?>
-
```
Comment (spflueger) [inline examples/python/D0ToKs0KpKm.xml:2]: why are these two xml files here in the first place. These are left overs from a test verifying the angular distribution

File: expertsystem/state/particle.py [hunk 3]
```diff
@@ -305,19 +305,21 @@ def to_dict(input_ordered_dict: OrderedDict) -> dict:
     name_label = Labels.Name.name
     with open(file_path, "rb") as xmlfile:
         full_dict = xmltodict.parse(xmlfile)
-        for particle_definition in full_dict["ParticleList"]["Particle"]:
-            particle_name = particle_definition[name_label]
-            particle_list[particle_name] = to_dict(particle_definition)
+        full_dict = full_dict.get("root", full_dict)
+    for particle_definition in full_dict["ParticleList"]["Particle"]:
+        particle_name = particle_definition[name_label]
+        particle_list[particle_name] = to_dict(particle_definition)
```
Comment (spflueger) [inline expertsystem/state/particle.py:311]: shouldn't this be indented to be inside the with scope?

File: examples/python/D0ToKs0KpKm.xml (hunk 2 above)
Comment (redeboer) [reply examples/python/D0ToKs0KpKm.xml:2]: True. But I don't want to address those (indeed malfunctioning) examples in this PR. An issue can be made for this

File: expertsystem/state/particle.py (hunk 3 above)
Comment (redeboer) [reply expertsystem/state/particle.py:311]: Why?

File: examples/python/D0ToKs0KpKm.xml (hunk 2 above)
Comment (spflueger) [reply examples/python/D0ToKs0KpKm.xml:2]: but cant this be just deleted instead of reformatting the whole file and an deleting it afterwards again?

Comment (spflueger) [reply examples/python/D0ToKs0KpKm.xml:2]: if you want this pr to be clean then I would suggest to apply a separate chore PR before that

File: expertsystem/state/particle.py (hunk 3 above)
Comment (spflueger) [reply expertsystem/state/particle.py:311]: Oh i thought with would create a scope, but it doesn't. Nvm then

File: examples/python/D0ToKs0KpKm.xml (hunk 2 above)
Comment (redeboer) [reply examples/python/D0ToKs0KpKm.xml:2]: In that case we'd better completely remove those examples

Comment (redeboer) [reply examples/python/D0ToKs0KpKm.xml:2]: and I'm not sure if we really want that, that's why I say, better make an issue for this

Comment (redeboer) [reply examples/python/D0ToKs0KpKm.xml:2]: Will be rebased onto #101

**Pull request merged**